include/fluent-bit/flb_record_accessor.h

```diff
@@ -34,6 +34,7 @@ struct flb_record_accessor {
     struct mk_list _head;        /* Head to custom list (only used by flb_mp.h) */
 };
 
+int flb_ra_subkey_num(struct flb_record_accessor *ra);
```

Comment:
Could you replace `num` with `count` everywhere? that would be much clearer.

Reply:
Thank you for reviewing.
I pushed following commit.
https://github.com/fluent/fluent-bit/pull/5590/commits/c9ece8302b5c3a72f3c1743a820a7b0470651292